templates/default/edit-and-start.sh.erb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
I think I'm missing something. Is this script called anywhere? Where does it get all the parameters?

Reply:
This script is for hopsworks-cloud. It will be called by hopsworks when a new VM is created using JClouds to change the IP, passwords, etc of config.ini so that kagent can connect to Hopsworks. I put it in this PR as prep work for the cloud version.